navigation-timing/resources/respond_slowly.py

```diff
@@ -0,0 +1,31 @@
+import time
+
+
+def main(request, response):
+    head = b"""<script>
+    let navBeforeLoadFinished = performance.getEntriesByType('navigation')[0];
```

Comment:
I don't think you need to call getEntriesByType multiple times. IIRC, you can call it once and then check its values in the different milestones

Reply:
Done